common/main/java/com/couchbase/lite/internal/core/C4Database.java

```diff
@@ -93,9 +91,6 @@ void nCopy(
         @NonNull
```

Comment:
These are a couple of other functions that are only used by tests.  I took the opportunity to move them to the TestUtils and out of the release code.